frontend/app/utils/validate.shared.ts

```diff
@@ -50,8 +62,14 @@ export const widgetFieldsSchema = z.object({
     .max(WIDGET_FONT_SIZES.max, widgetFontSizeError),
   widgetTitleText: z
     .string()
-    .min(1, { message: 'Widget title cannot be empty' }),
-  widgetDescriptionText: z.string().optional(),
+    .min(1, { message: 'Widget title cannot be empty' })
+    .max(WIDGET_TITLE_MAX_LENGTH, { message: 'Widget title is too long' }),
```

Comment:
Users know title for what from the page they're on.
```suggestion
    .max(WIDGET_TITLE_MAX_LENGTH, { message: 'Title is too long' }),
```